tools/ci_build/github/linux/docker/scripts/install_dotnet.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -e -x
+
+if [ -f /etc/redhat-release ]; then
+    dnf update --refresh -y \
+    && dnf install -y dotnet-sdk-6.0
+elif [ -f /etc/os-release ]; then
+  # Get Ubuntu version
+  declare repo_version=$(if command -v lsb_release &> /dev/null; then lsb_release -r -s; else grep -oP '(?<=^VERSION_ID=).+' /etc/os-release | tr -d '"'; fi)
```

Comment:
Since we control the image, we can install lsb_release to the image if needed. Then this code could be simpler.